.ci/windows/msbuild-tools.ps1

```diff
@@ -3,6 +3,6 @@
 #
 
 # Install visualstudio2019buildtools
-& choco install visualstudio2019buildtools -m -y --no-progress --force -r --version=16.4.5.0
-& choco install visualstudio2019professional -m -y --no-progress --force -r --version=16.4.5.0 --package-parameters "--includeRecommended --includeOptional"
+& choco install visualstudio2019buildtools-preview -m -y --no-progress --force -r --version=16.8.0.60000-preview1
+& choco install visualstudio2019professional-preview -m -y --no-progress --force -r --version=16.8.0.60000-preview1 --package-parameters "--includeRecommended --includeOptional"
```

Comment:
See https://chocolatey.org/packages/visualstudio2019buildtools-preview/16.8.0.60000-preview1 and https://chocolatey.org/packages/visualstudio2019professional-preview/16.8.0.60000-preview1

Reply:
It failed to install the tools

```
[2020-11-09T16:49:16.091Z] visualstudio2019buildtools-preview has been installed.
[2020-11-09T16:49:16.352Z]   visualstudio2019buildtools-preview may be able to be automatically uninstalled.
[2020-11-09T16:49:16.352Z]  The install of visualstudio2019buildtools-preview was successful.
[2020-11-09T16:49:16.352Z]   Software installed to '"C:\Program Files (x86)\Microsoft Visual Studio\Installer"'
[2020-11-09T16:49:16.352Z] 
[2020-11-09T16:49:16.352Z] Chocolatey installed 1/1 packages. 
[2020-11-09T16:49:16.352Z]  See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
[2020-11-09T16:49:16.931Z] visualstudio2019buildtools-preview installation failed. See the logs.
```

So we might need to debug what's the root reason

Reply:
I think VS tools are needed only for the Microsoft.WebApplication.targets file for the AspNetFullFrameworkSampleApp? Running `dotnet build` on a PC that has only the .NET 5 SDK installed, this is the only thing that fails to build.